walkeeper/src/timeline.rs

```diff
@@ -1,50 +1,85 @@
-//! This module contains tools for managing timelines.
-//!
+//! This module contains timeline id -> safekeeper state map with file-backed
+//! persistency and support for interaction between sending and receiving wal.
```

Comment:
nitpick: persistence

Reply:
wow, didn't actually know the difference